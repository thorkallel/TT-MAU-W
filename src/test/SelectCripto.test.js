import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

import CriptoState from "../context/cripto/criptoState";
import CotizacionState from "../context/cotizacion/cotizacionState";

import SelectCripto from "../hooks/useCriptomoneda";

describe("Pruebas en <SelectCripto />", () => {
  test("debe mostrar <SelectCripto /> correctamente", () => {

    const wrapper = shallow(
      <SelectCripto />
    );

    expect(wrapper).toMatchSnapshot();
  });

	test("debe mostrar los items <SelectCripto />", () => {

    const wrapper = shallow(
      <SelectCripto />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
