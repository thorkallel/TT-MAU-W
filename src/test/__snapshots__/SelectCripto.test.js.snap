// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <SelectCripto /> debe mostrar <SelectCripto /> correctamente 1`] = `
Array [
  <Component />,
  <Component />,
  <Component />,
]
`;

exports[`Pruebas en <SelectCripto /> debe mostrar los items <SelectCripto /> 1`] = `
Array [
  <Component />,
  <Component />,
  <Component />,
]
`;
